ember.js/2016-06/june-11-12_F2F.md

```diff
@@ -0,0 +1,659 @@
+- [ ] Create a SUBTEAM doc or guide.
+  - [ ] Suggest that notes be stored in core-notes
+  - [ ] Suggest that after notes are created a link is posted into `#core-notes` on Slack
+- [ ] New subteam: *Performance Team*
+  - [ ] Kris will create, liason. Goal of this team is to automate and make following good perf practices easy.
+  - [ ] Kris will propose members
+- [ ] New subteam: **Render Team**
+  - [ ] wycats will create, liason
+  - [ ] wycats will propose members
+- [ ] Module name strike team
+  - [ ] create a charter via the RFC process
+  - [ ] wycats wants ed/dan on it to manage constraints "square the circle" ;-) Ed is foolish enough to volunteer.
+  - [ ] Current progress https://github.com/emberjs/ember.js/pull/13022
+- [ ] Security team strike team
+  - [ ] create a charter via the RFC process
+  - [ ] EF will become the security contact
+  - [ ] EF will take up a strike team to improve the process (https://hackerone.com/)
+
+### Committers
+
+YK: Other projects have different team structures, perhaps we should consider two things:
+
+- path to core is a poor term. Perhaps "committers" (cargo culting the rails term)
+- the second circle of people, what we have called path to core, should likely be widened and allow those people access to #core
+
+YK: One of the issues with expanding #core a lot is that we have to pay for Slack, and it's currently funded out of donations.
+
+(Several people expressed a lack of clarity on how companies should contribute to Ember directly)
+
+LS: The most significant financial contribution to Ember is hiring people, and that is fairly decentralized
+
+YK: I really like the fact that this significant financial contribution doesn't need to go through a central decision making process, but that hiring someone doesn't automatically mean that that person is on the core team or has any special authority (of course, hiring someone will give them a lot of time, and if that time is used well, they'll earn up quickly).
+
+LS: Other contributions are fairly scattered and case-by-case.
+
+(Questions about how to fund something like Slack)
+
+YK: if you have a thing to do that is focused and bounded, Tilde will manage the accounting and logistics. There are no concerns about Linkedin etc paying for SauceLabs, but to the extent that we can have Tilde do the busy-work and invoice LinkedIn, it'll be easier to manage all of the accounts, authentication, etc. in one place.
+
+YK: Quite often, the process of getting a big company to pay for something directly is more cumbersome than invoicing the big company, so we should do that if possible.
+
+(People felt that would work very well)
+
+Action items (proposed!!! Really needs a proposal doc):
+
+- [ ] Do it! "Committers"
+- [ ] Access to #core #core-ot
+- [ ] Options for leaving:
+  - [ ] Yearly opt-in to renew your status in the next year's class "Class of 2017"
+    - [ ] Remind them that the acct costs $100 a year (or whatever)
```

Comment:
We'd discussed making sure this process involved a couple different options up front, like "you can bill my company, they'll cover it" (which some people won't otherwise think of) and "I need scholarship for this." 

Definitely don't want this to be a meaningful barrier to entry for potential committers, just a means of helping us scale up that will most likely be covered most times by generous employers for whom it's insignificant.